diff --git a/lib/utils/background_push.dart b/lib/utils/background_push.dart
index 02455d3..768f444 100644
--- a/lib/utils/background_push.dart
+++ b/lib/utils/background_push.dart
@@ -43,7 +43,7 @@ import '../config/app_settings.dart';
 import '../widgets/matrix.dart';
 import 'platform_infos.dart';

-//<GOOGLE_SERVICES>import 'package:fcm_shared_isolate/fcm_shared_isolate.dart';
+import 'package:fcm_shared_isolate/fcm_shared_isolate.dart';

 class NoTokenException implements Exception {
   String get cause => 'Cannot get firebase token';
@@ -69,7 +69,7 @@ class BackgroundPush {
   final pendingTests = <String, Completer<void>>{};
   bool firebaseEnabled = false;

-  //<GOOGLE_SERVICES>final firebase = FcmSharedIsolate();
+  final firebase = FcmSharedIsolate();

   DateTime? lastReceivedPush;

@@ -92,7 +92,7 @@ class BackgroundPush {
   }

   void _init() async {
-    //<GOOGLE_SERVICES>firebaseEnabled = true;
+    firebaseEnabled = true;
     try {
       mainIsolateReceivePort?.listen((message) async {
         try {
@@ -128,18 +128,18 @@ class BackgroundPush {
       }
       await initialiseLocalNotifications();
       Logs().v('Flutter Local Notifications initialized');
-      //<GOOGLE_SERVICES>firebase.setListeners(
-      //<GOOGLE_SERVICES>  onMessage: (message) => PushHelper.pushHelper(
-      //<GOOGLE_SERVICES>    PushNotification.fromJson(
-      //<GOOGLE_SERVICES>      Map<String, dynamic>.from(message['data'] ?? message),
-      //<GOOGLE_SERVICES>    ),
-      //<GOOGLE_SERVICES>    clients: clients,
-      //<GOOGLE_SERVICES>    l10n: l10n,
-      //<GOOGLE_SERVICES>    activeRoomId: matrix?.activeRoomId,
-      //<GOOGLE_SERVICES>    activeClient: clients.firstOrNull,
-      //<GOOGLE_SERVICES>    flutterLocalNotificationsPlugin: _flutterLocalNotificationsPlugin,
-      //<GOOGLE_SERVICES>  ),
-      //<GOOGLE_SERVICES>);
+      firebase.setListeners(
+        onMessage: (message) => PushHelper.pushHelper(
+          PushNotification.fromJson(
+            Map<String, dynamic>.from(message['data'] ?? message),
+          ),
+          clients: clients,
+          l10n: l10n,
+          activeRoomId: matrix?.activeRoomId,
+          activeClient: clients.firstOrNull,
+          flutterLocalNotificationsPlugin: _flutterLocalNotificationsPlugin,
+        ),
+      );
       if (Platform.isAndroid) {
         await UnifiedPush.initialize(
           onNewEndpoint: _newUpEndpoint,
@@ -198,7 +198,7 @@ class BackgroundPush {
     required Client client,
   }) async {
     if (PlatformInfos.isIOS) {
-      //<GOOGLE_SERVICES>await firebase.requestPermission();
+      await firebase.requestPermission();
     }
     if (PlatformInfos.isAndroid) {
       _flutterLocalNotificationsPlugin
@@ -387,7 +387,7 @@ class BackgroundPush {
     Logs().v('Setup firebase');
     if (_fcmToken?.isEmpty ?? true) {
       try {
-        //<GOOGLE_SERVICES>_fcmToken = await firebase.getToken();
+        _fcmToken = await firebase.getToken();
         if (_fcmToken == null) throw ('PushToken is null');
       } catch (e, s) {
         Logs().w('[Push] cannot get token', e, e is String ? null : s);
@@ -522,8 +522,8 @@ class BackgroundPush {
     Logs().i('[Push] UnifiedPush using endpoint $endpoint');
     final oldTokens = <String?>{};
     try {
-      //<GOOGLE_SERVICES>final fcmToken = await firebase.getToken();
-      //<GOOGLE_SERVICES>oldTokens.add(fcmToken);
+      final fcmToken = await firebase.getToken();
+      oldTokens.add(fcmToken);
     } catch (_) {}
     final client = clientFromInstance(i, clients) ?? clients.first;
     await setupPusher(
diff --git a/pubspec.yaml b/pubspec.yaml
index 9b1952f..435bdba 100644
--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -33,7 +33,7 @@ dependencies:
   dio: ^5.9.2
   dynamic_color: ^2.1.0
   emojis: ^3.2.0
-  #fcm_shared_isolate: ^0.2.0
+  fcm_shared_isolate: ^0.2.0
   file_picker: ^12.0.0
   file_selector: ^1.1.0
   flutter:
